lms/static/js/fixtures/instructor_dashboard/batchenrollment.html

```diff
@@ -0,0 +1,71 @@
+<section id="membership" class="idash-section active-section" aria-labelledby="header-membership">
+    <h2 class="hd hd-2" id="header-membership">Membership</h2>
+
+    <fieldset class="batch-enrollment membership-section">
+        <legend id="heading-batch-enrollment" class="hd hd-3">Batch Enrollment</legend>
+        <label>
+            Enter email addresses and/or usernames separated by new lines or commas.
+            You will not get notification for emails that bounce, so please double-check spelling.
```

Comment:
This is just a fixture and contains content from previous changes so its out of scope of this story.